### ISSUE TITLE
Output files missing

### ISSUE THREAD

**Issue by pshah0086** [mdbloice/Augmentor#232]
I tried using Datapipeline to generated augmented images along with its mask. But even after successfull execution I cannot see any output folder in neither my image directory nor my labels directeory. i also tried uncommenting Pipeline.py output directory member variable and specifing output directory path. But none of the method worked. i simply cannot see my augmented images and masks anywhere. 
Irony is i can visualize augmented images with its mask in Jupyter notebook using "imshow", it means augmentor worked and has generated images but i simply can't see it anywhere on my pc.

Please help if someone else also faced this issue. Thanks.

**Comment by pratikkulkarni228** [comment]
Yeah, I am facing the same issue. Inspite of providing an output_directory to the Pipeline constructor, it saves the outputs in the one directory previous to the source directory.
